### PR TITLE
updating charmhub for v2.0.0

### PR DESCRIPTION
**Pull request by MiaAltieri**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -61,19 +61,19 @@ jobs:
...
+        uses: canonical/charming-actions/channel@2.0.0-rc
         id: channel
       - name: Upload charm to charmhub
-        uses: canonical/charming-actions/upload-charm@1.0.3
+        uses: canonical/charming-actions/upload-charm@2.0.0-rc
         with:
           credentials: "${{ secrets.CHARMHUB_TOKEN }}"
           github-token: "${{ secrets.GITHUB_TOKEN }}"
-          channel: "dpe/edge"
+          channel: "${{ steps.channel.outputs.name }}"
```
Comment (jnsgruk) [inline .github/workflows/release.yaml:79]: Did you mean to do this? :)

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -61,19 +61,19 @@ jobs:
       - lib-check
       - lint
       - unit-test
-      - integration-test-lxd
+      - integration-test
```
Comment (jnsgruk) [inline .github/workflows/release.yaml:64]: Copy/paste?

Comment (MiaAltieri) [reply .github/workflows/release.yaml:64]: shameless copy paste, thanks for catching this 🤦‍♀️

File: .github/workflows/release.yaml (hunk 1 above)
Comment (MiaAltieri) [reply .github/workflows/release.yaml:79]: oof, thanks for catching this

**Review by zmraul — APPROVED** [review]

**Review by delgod — APPROVED** [review]

**Pull request merged**